src/deprecated.jl

```diff
@@ -1,5 +1,49 @@
 import Base: @deprecate
 
+@deprecate DataFrame(t::Type, nrows::Integer, ncols::Integer) DataFrame(fill(t, ncols), nrows)
+
+@deprecate DataFrame(column_eltypes::AbstractVector{<:Type}, cnames::AbstractVector{Symbol},
+                     nrows::Integer;
+                     makeunique::Bool=false) DataFrame(AbstractVector[elty >: Missing ?
+                                                                      fill!(Tables.allocatecolumn(elty, nrows), missing) :
```

Comment:
`Tables.allocatecolumn` isn't available from user code. Maybe just use `Vector` and `missings`?

Reply:
I was thinking about this and the use of `Index` below. OK - I will change both.

Reply:
I have pushed something that is shorter and should be equivalent (hopefully).